e2e/cephfs_helper.go

```diff
@@ -24,7 +24,7 @@ func validateSubvolumegroup(f *framework.Framework, subvolgrp string) error {
 	cmd := fmt.Sprintf("ceph fs subvolumegroup getpath myfs %s", subvolgrp)
 	stdOut, stdErr, err := execCommandInToolBoxPod(f, cmd, rookNamespace)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to exec command in toolbox %w", err)
```

Comment:
I think it is always good to split error reporting with a `:`, like:

```go
fmt.Errorf("failed to exec command in toolbox: %w", err)
```

This also makes `with error` unneeded in other new error returns.